docs/pages/User/GettingStarted/uninstall.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Uninstall Liqo
+weight: 4
+---
+
+## Uninstall steps
+
+This procedure uninstalls Liqo on your cluster.
+
+```bash
+export UNINSTALL=1
```

Comment:
I feel "export" variables are a pain and I suggest to avoid them.
I would prefer to rename the script and call it with an additional parameter:
install.sh ==> liqo-setup.sh
==> liqo-setup.sh install (to install everything)
==> liqo-setup.sh remove (to remove everything)
@palexster Your opinion?

Reply:
Concerning the uninstall script, I totally agree since we are asking for a complete different behaviour that should be made explicit as much as possible. We can still preserve the one-liner and no-clone policy by using an approach like [that](https://stackoverflow.com/questions/4642915/passing-parameters-to-bash-when-executing-a-script-fetched-by-curl). I would suggest something like:

```
curl https://raw.githubusercontent.com/LiqoTech/liqo/master/liqo-setup.sh | bash -s -- unistall
```

For this PR, I suggest to make install.sh with an "option" interface. Due to the implications of renaming to CI and documentation, I would rather do this in a dedicated back-to-back PR.